### ISSUE TITLE
retards using this to grab stupid kids

### ISSUE THREAD

**Issue by cheatdeveloper73** [NightfallGT/Mercurial-Grabber#35]
lmao this is the funniest shit ever, all the issues are just some braindead 10 year old skids trying to log people. I just forwarded some guy's info to the fbi after he tried to get me with this. Your anti-vm & anti-debug shit doesn't even work. and if it did you can just statically analyze it, that's how retardedly stupid this is.

**Comment by lachiyac** [comment]
weren't you the one that said this "cant find cookie
Roblox Cookie
Unable to find cookie from Roblox Studio registry
Mercurial Grabber | github.com/nightfallgt/mercurial-grabber"
sounds like you fit your own definition perfectly, stupid little kids using it to log people and cant get it to work

**Comment by ghost** [comment]
there fucking stupid that why some kid try to send this to me and IDA pro is my love!!

**Comment by AizonF** [comment]
yeah man, nightfallgt was nice enough to make a open source grabber so other people could check out source, and skids are just trying to log people with it 🤣

**Comment by ghost** [comment]
you don't even need statical analysis, string scan just does the trick lmao
